encord/workflow/stages/consensus_review.py

```diff
@@ -82,6 +108,28 @@ class ConsensusReviewTask(WorkflowTask):
     data_title: str
     options: List[ConsensusReviewOption]
 
+    """
+    Tasks in the Review stage of a Consensus Project.
+
+    **Params**
+
+    - assignee: User assigned to a task.
+    - data_hash: Unique ID for the data unit.
+    - data_title: Name of the data unit.
+    - options: Specify the labels for the task.
+
+    Allowed actions:
+
+    - approve: Approves a task
+    - reject: Rejects a task.
+    - assign: Assigns a task to a user.
+    - release: Releases a task from the current user.
+
+    **Returns**
```

Comment:
Same as above, `Returns` make sense for a function, not for a class

Reply:
Updated